app/application.js

```diff
@@ -1,26 +1,33 @@
 "use strict";
 
-var root = 'http://localhost:4000/api';
-var origin = window.location.origin;
+const root = 'http://localhost:4000/api';
+const origin = window.location.origin;
+
+function link(id, text){
+  return `<a href= ${origin}/announcement.html?id=${id}>${text}<br/><br/></a>`
+}
```

Comment:
the only way i could get it to know what `link` was was by taking it out of the var App